v0/pipeline.json

```diff
@@ -44187,6 +44187,9 @@
                     "minLength" : 1
                   } ]
                 },
+                "applicationRegex" : {
+                  "type" : "string"
```

Comment:
do we need minLength here?
If we default to empty string here in UI, the following condition for one of "applicationsList" or "applicationRegex" to be required will always be true

Reply:
On UI we should implement with removing field completely of whichever one's radio is not checked @litvinovstas